release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1004,6 +1004,11 @@ Bound service account tokens are audience-bound and time-bound. For more informa
 
 Additionally, the kubelet refreshes tokens automatically after they reach 80% of duration, and `client-go` watches for token changes and reloads automatically. The combination of these two behaviors means that most usage of bound tokens is no different from usage of legacy tokens that never expire. Non-standard usage outside of `client-go` might cause issues.
 
+[IMPORTANT]
+====
+If you use a custom SCC that restricts volume usage in the `volumes` field, you might need to add `projected` to the list of volume types allowed by that SCC. This is because every pod is now injected with a projected volume.
```

Comment:
@stlaz @s-urbaniak I have it as that they "might need to add projected", but is it certain enough that we can say "you must add projected"?